README.md

```diff
@@ -1 +1 @@
-# spark-mhealth-demo
\ No newline at end of file
+# spark-mhealth-demo
```

Comment:
Some files are getting touched by pre-commit  -- usually to edit white space or number of lines at end of file.